archetypes/Pools/PoolDetailsModal/index.tsx

```diff
@@ -21,37 +21,40 @@ type PoolProps = {
 };
 
 export default (({ open, onClose, poolDetails, network }) => {
-    const { name, leverage, keeper, committer, collateralAsset, collateralAssetAddress } = poolDetails || {};
-
-    const formatAddress = (addr: string) => `${addr?.slice(0, 4)}...${addr?.slice(40, 42)}`;
-
-    const poolDetailsData = [
-        { name: 'Pool Ticker', value: name },
-        {
-            name: 'Price Feed',
-            value: name?.split('-')[1],
-            href: getPriceFeedUrl(name),
-        },
-        {
-            name: 'Power Leverage',
-            value: leverage,
-        },
-        {
-            name: 'Collateral Asset',
-            value: collateralAsset,
-            href: constructExplorerLink(BlockExplorerAddressType.token, collateralAssetAddress, network),
-        },
-        {
-            name: 'Deployer',
-            value: formatAddress(committer),
-            href: constructExplorerLink(BlockExplorerAddressType.address, committer, network),
-        },
-        {
-            name: 'Keeper Contract',
-            value: formatAddress(keeper),
-            href: constructExplorerLink(BlockExplorerAddressType.address, keeper, network),
-        },
-    ];
+    const { name, leverage, keeper, committer, collateralAsset, collateralAssetAddress } = poolDetails;
+
+    const formatAddress = useCallback((addr: string) => `${addr?.slice(0, 4)}...${addr?.slice(40, 42)}`, []);
+
+    const poolDetailsData = useMemo(
+        () => [
+            { name: 'Pool Ticker', value: name },
+            {
+                name: 'Price Feed',
+                value: name?.split('-')[1],
+                href: getPriceFeedUrl(name),
+            },
+            {
+                name: 'Power Leverage',
+                value: leverage,
+            },
+            {
+                name: 'Collateral Asset',
+                value: collateralAsset,
+                href: constructExplorerLink(BlockExplorerAddressType.token, collateralAssetAddress, network),
+            },
+            {
+                name: 'Deployer',
+                value: formatAddress(committer),
+                href: constructExplorerLink(BlockExplorerAddressType.address, committer, network),
```

Comment:
not part of this change set but not sure what this is supposed to be (deployer or committer)

Reply:
probably deployer, but is showing committer